metadata-service/factories/src/main/java/com/linkedin/gms/factory/auth/NativeUserServiceFactory.java

```diff
@@ -0,0 +1,58 @@
+
+
+package com.linkedin.gms.factory.auth;
+
+import com.datahub.authentication.user.NativeUserService;
+import com.linkedin.entity.client.JavaEntityClient;
+import com.linkedin.gms.factory.spring.YamlPropertySourceFactory;
+import com.linkedin.metadata.entity.EntityService;
+import com.linkedin.metadata.secret.SecretService;
+import javax.annotation.Nonnull;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.Scope;
+
+
+@Configuration
+@PropertySource(value = "classpath:/application.yml", factory = YamlPropertySourceFactory.class)
+public class NativeUserServiceFactory {
+  /**
+   * +  @Inject
+   * +  @Named("entityService")
+   * +  private EntityService _entityService;
+   * +
+   */
+  @Autowired
+  @Qualifier("entityService")
+  private EntityService _entityService;
+
+  /**
+   * +  @Inject
+   * +  @Named("javaEntityClient")
+   * +  private JavaEntityClient _javaEntityClient;
+   * +
+   */
+  @Autowired
+  @Qualifier("javaEntityClient")
+  private JavaEntityClient _javaEntityClient;
+
+  /**
```

Comment:
What's this stuff doing?

Reply:
This is the same pattern used across other factories in our codebase

Reply:
i mean the comments